### ISSUE TITLE
Not working when no songs in playlist

### ISSUE THREAD

**Issue by Nordiii** [BlitzKraig/fvtt-SoundBoard#8]
In case you have no songs in the playlist the SoundBoard wont play and just throw this error:

```
soundboardaudiohelper.js:33 Uncaught (in promise) TypeError: Cannot read property 'createGain' of null
    at SBAudioHelper.play (soundboardaudiohelper.js:33)
    at Function.playSound (soundboard.js:114)
    at Function.playSoundOrStopLoop (soundboard.js:89)
    at HTMLButtonElement.onclick (game:1)
```

FoundryVTT 0.7.5
Current SoundBoard
Current dnd5e

**Comment by BlitzKraig** [comment]
I have also received reports of an issue if the user does not have a scene set up, though have not tested yet.
Adding here as a comment to remind me to look into it with this ticket.

**Comment by BlitzKraig** [comment]
See #23

**Comment by BlitzKraig** [comment]
Fixed in 1.2
